src/server/ApiRoutes/Search.js

```diff
@@ -57,14 +57,34 @@ function fetchResults(searchKeywords = '', page, sortBy, order, field, filters,
     query: { q: searchKeywords, sortBy, order, field, filters },
   };
 
+  let drbRequesting = true;
+  const drbPromise = appConfig.features.includes('drb-integration') ?
+    Promise.race([
+      new Promise((resolve) => {
+        setTimeout(() => {
+          if (!drbRequesting) return false;
```

Comment:
What's the purpose of `drbRequesting`? Can't you just rely on `Promise.race` to only return the first of the resolved values? If `ResearchNow.search` resolves first, it will be returned by `Promise.race`; When your `setTimeout` fires later, it's okay that it resolves `[]` because `Promise.race` has already resolved the real drb results

Reply:
When I tried implementing this without `drbRequesting`, I got `Drb timeout` in the logs for each request, even when there was no timeout.